CFG.md

```diff
@@ -0,0 +1,55 @@
+# Sonar-PHP Control Flow Graph
+
+----
+## Specification
+
+A CFG is a graph of [basic blocks](https://en.wikipedia.org/wiki/Basic_block) (from now on, we will use the term 'block').
+
+### Blocks
+
+A block has the following attributes:
+
+* successors - blocks that are executed after
+* predecessors - blocks that are executed before
+* syntactic successor - if no jump is applied, the block that immediately follows
```

Comment:
What do you mean by 'if no jump is applied'? Do you mean that there is no jump in the elements of the block?